pom.xml

```diff
@@ -176,6 +176,7 @@ governing permissions and limitations under the License. -->
                             <includes>
                                 <include>**/*Tests.class</include>
                                 <include>**/*Test.class</include>
+                                <include>**/Test*.class</include>
```

Comment:
This means we need to change that in elasticsearch-parent as well to be consistent.